gtfs-realtime/spec/en/reference.md

```diff
@@ -1,14 +1,29 @@
 A GTFS Realtime feed lets transit agencies provide consumers with realtime information about disruptions to their service (stations closed, lines not operating, important delays, etc.) location of their vehicles, and expected arrival times.
 
-Version 1.0 of the feed specification is discussed and documented on this site.
+Version 2.0 of the feed specification is discussed and documented on this site.
 
 ### Term Definitions
 
-*   **required**: Exactly one
-*   **repeated**: Zero or more
```

Comment:
> Do we leave this as the "Cardinality" column and change the name of the column for the new semantic fields to "Requirement", or something similar?

I think that's the best approach. My reasoning:
- Cardinality best describes the number of entities allowed in that field, so let's keep it purely to numbers. Of course the savvy user may determine the "requirement" from the number, but I think it's useful to explicitly state the "requirement" in a separate column.
- A new column, `Required` or something similar makes it very clear that the user should check that column to see if the field is needed. 

I think it's good to separate the two concepts; a) is it needed (check `Required` column) and 2) how many can I provide (check `Cardinality` column).

I agree about `repeated`; it's not user friendly. If you have the `Cardinality` column, then the `Required` column entries become: Yes (for `required`) and No (for `optional` and `repeated`).